### PR TITLE
feat(i18n): import strings from otp-ui packages

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
This PR adds support for importing language strings from otp-ui. Strings copied from the repo have been removed. The deepmerge dependency is new in `package.json`, but not in `yarn.lock` (it was previously a sub dependency)

### CODE REVIEW

File: lib/util/i18n.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import flatten from 'flat'
 
+const merge = require('deepmerge')
```
Comment (binh-dam-ibigroup) [inline lib/util/i18n.js:3]: Can this be a regular `import`?

File: lib/util/i18n.js [hunk 2]
```diff
@@ -45,6 +47,33 @@ export function getConfigLocales(configLanguages) {
...
+ * Loads in all localized strings from @opentripplanner packages.
+ * Package list is loaded from the package.json and each yml file then imported.
+ *
+ * All strings are then merged
+ */
+async function loadOtpUiLocaleData(matchedLocale) {
+  const packageJson = await import('../../package.json')
+  const otpUiMessages = await Promise.all(
+    Object.keys(packageJson.dependencies)
+      .filter((pkg) => pkg.includes('@opentripplanner'))
```
Comment (binh-dam-ibigroup) [inline lib/util/i18n.js:60]: `.startsWith`?

File: lib/util/i18n.js [hunk 3]
```diff
@@ -76,9 +105,12 @@ export async function loadLocaleData(matchedLocale, customMessages) {
       break
   }
 
+  const otpUiMessages = await loadOtpUiLocaleData(matchedLocale)
+
   // Merge custom strings into the standard language strings.
   const mergedMessages = {
     ...flatten(messages),
+    ...flatten(otpUiMessages),
```
Comment (binh-dam-ibigroup) [inline lib/util/i18n.js:113]: There might be a case where we would need OTP-RR messages to overwrite default OTP-UI messages, so ideally `otpUiMessages` should be assigned before `messages`.

**Review by binh-dam-ibigroup — APPROVED** [review]
Nice! See the few minor comments.

File: lib/util/i18n.js (hunk 1 above)
Comment (miles-grant-ibigroup) [reply lib/util/i18n.js:3]: Sadly no: https://github.com/TehShrike/deepmerge#include

File: lib/util/i18n.js (hunk 2 above)
Comment (miles-grant-ibigroup) [reply lib/util/i18n.js]: Good call! Refactored in bfe701697e8cbceef1b004a198352f7504f4e6b0

File: lib/util/i18n.js (hunk 3 above)
Comment (miles-grant-ibigroup) [reply lib/util/i18n.js:112]: That feels good! Refactored in bfe701697e8cbceef1b004a198352f7504f4e6b0

File: lib/util/i18n.js (hunk 1 above)
Comment (binh-dam-ibigroup) [reply lib/util/i18n.js:3]: Place a comment in the code referencing that link.

Comment (miles-grant-ibigroup) [reply lib/util/i18n.js:4]: Done!

**Review by philip-cline — APPROVED** [review]

**Pull request merged**